### PR TITLE
Add Nuimo.app v1.7.0

### PR DESCRIPTION
**Pull request by anthonyangel**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/nuimo.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'nuimo' do
+  version '1.7.0'
+  sha256 'e50c8f751b6944bdbe64f1c01e61d9c4d939fd63850bd4d112883aa46db10f3b'
+
+  url 'https://files.senic.com/nuimo-app-osx-1-7-0.zip'
```
Comment (vitorgalvao) [inline Casks/nuimo.rb]: Changed this for interpolation

File: Casks/nuimo.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'nuimo' do
+  version '1.7.0'
+  sha256 'e50c8f751b6944bdbe64f1c01e61d9c4d939fd63850bd4d112883aa46db10f3b'
+
+  url "https://files.senic.com/nuimo-app-osx-#{version.dots_to_dashes}.zip"
```
Comment (vitorgalvao) [inline Casks/nuimo.rb]: Changed this for interpolation.

Comment (vitorgalvao) [inline Casks/nuimo.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md). Address is `https://files.senic.com/nuimo-app-osx-appcast.xml`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by fanquake** [comment]
Travis failure:
```
>>> brew cask style Casks/nuimo.rb
== Casks/nuimo.rb ==
C:  5: 77: Trailing whitespace detected.
1 file inspected, 1 offense detected
```

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**